README.md

```diff
@@ -112,6 +112,20 @@ Declaring these resources will configure `device.conf` and apply the base class
 puppet device --verbose --target bigip.example.com
 ```
 
+#### Signing certificates on first run
+
+If this is the first time executing against this device and the certificate has not yet been signed, the following may be seen:
+```bash
+Notice: Did not receive certificate
+```
+
+Execute the following to sign the certificates for the devices:
+```bash
+puppet cert sign --all
```

Comment:
`puppet cert sign --all` is unsafe, since any other pending certificate requests will be signed.

Reply:
Good point.

I've updated with just the hostname, or will I remove this section entirely and assume the target audience has enough knowledge of Puppet?

Reply:
I would think a seasoned user would know, but who's to say a newer user would explore with a device? Seems good with a hostname.